.travis.yml

```diff
@@ -49,6 +53,24 @@ install:
 
   # install TensorFlow (CPU version).
   - pip install tensorflow
+  
+  # install cntk
+  - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
+      wget http://cntk.ai/PythonWheel/ForKeras/CPUOnly/cntk-2.0rc3-cp27-cp27mu-linux_x86_64.whl;
+      pip install cntk-2.0rc3-cp27-cp27mu-linux_x86_64.whl;
```

Comment:
Yes, I should update this install command. The wheel under "ForKeras" section is created 2 days before CNTK 2.0 GA release, so at that time, we make this internal build public to make it work with keras travis ci pipeline. I will update it to use the official CNTK 2.0 build. Thanks for point this out.